home_work/hw_9/candles/candles.py

```diff
@@ -0,0 +1,35 @@
+"""
+Module for burning candles.
+"""
+
+
+def burn_candles(candles, candles_end):
+    """
+    Calculate the total number of candles burned.
+    """
+
+    fired_candles = candles
+
+    while candles >= candles_end:
+        new_candles = int(candles / candles_end)
+        fired_candles += new_candles
+        candles = new_candles + (candles % candles_end)
+    return fired_candles
+
+
+if __name__ == '__main__':
+    print(burn_candles(5, 2))
```

Comment:
you do not need to print anything here, but if you want to log something use logger module 
https://docs.python.org/3/library/logging.html